api/transactions.js

```diff
@@ -607,7 +570,7 @@ function getAccountTransactions(api, options, callback) {
 
   function sortTransactions(transactions, async_callback) {
     transactions.sort(function(first, second) {
-      return compareTransactions(first, second, options.earliestFirst);
+      return utils.compareTransactions(first, second, !options.earliestFirst);
```

Comment:
`var compare = options.earliestFirst ? utils.compareTransactions : _.rearg(utils.comareTransactions, 1, 0);`
`transactions.sort(compare);`
could be used to eliminate some redundancy in `compareTransactions`